packages/components/src/TextField/TextField.tsx

```diff
@@ -78,7 +78,7 @@ export const TextField = ({
         startIconAdornment={icon}
         endIconAdornment={
           status === 'success' && (
-            <div className={classnames(styles.icon, styles.success, disabled && styles.disabled)}>
+            <div className={classnames(styles.icon, styles.success)}>
```

Comment:
Was this meant to be in this PR?

Reply:
Yeah, strangely when I removed the animation thing from TextField, it must have been masking this issue where the icon was getting double 0.3 opacity applied to it.

Reply:
Ah right, gotcha 🤔 I suppose this is on the icon anyhow so that was always weird

Reply:
Yeah so the container wrapping this has already got a disabled class which is getting 0.3 opacity applied